src/runner/patch-round.js

```diff
@@ -68,53 +68,36 @@ const roundFractionImpl = {
   },
 };
 
-const roundSignificantImpl = {
-  [DECIMAL_128](arg, significantDigits, mode) {
-    return arg.toSignificantDigits(significantDigits, mode);
-  },
-  [BIG_DECIMAL](arg, significantDigits, mode) {
-    return arg.prec(significantDigits, mode);
-  },
-};
-
-function round(decimal, options = Object.create(null)) {
+const round = (decimal, options = {}) => {
   // decimal argument is guaranteed to be either a Decimal or Big here
   const implName = decimal instanceof Decimal ? DECIMAL_128 : BIG_DECIMAL;
 
-  if (!options || typeof options !== "object") {
-    throw new TypeError("second argument to Decimal.round() must be an object");
+  if (typeof options !== "object") {
+    throw new TypeError("Second argument to Decimal.round() must be an object");
   }
 
-  let { maximumFractionDigits, maximumSignificantDigits, roundingMode } =
-    options;
+  let { maximumFractionDigits, roundingMode } = options;
 
-  if (roundingMode === undefined) {
-    throw new TypeError("roundingMode option is required for Decimal.round()");
+  if (!roundingMode) {
+    throw new TypeError("RoundingMode option is required for Decimal.round()");
   }
-  const internalMode = roundingModeImpl[implName](roundingMode);
 
-  if (maximumFractionDigits !== undefined) {
-    maximumFractionDigits = +maximumFractionDigits;
-    return roundFractionImpl[implName](
-      decimal,
-      maximumFractionDigits,
-      internalMode
-    );
-  }
+  const coercedFractionDigits = Number(maximumFractionDigits);
 
-  if (maximumSignificantDigits !== undefined) {
```

Comment:
The proposal does not currently support `maximumSignificantDigits` and I would rather keep it simple unless we hear a lot of requests for it.

Reply:
I agree that we shouldn't add things that are not in the proposal, but where are you seeing that it's not part of the proposal? There's no section header to link directly to, but it's in the [Common elements](https://github.com/tc39/proposal-decimal#common-elements) section if you scroll down a bit.

Reply:
Ughhhh it's not listed in the specific references. I'm honestly tempted to leave it out because I don't really see the obvious need for two slightly different ways to round. And if there is clamor for it in its absence we've leaved something. At least through the magic of version control, the code is still referenceable so we haven't wasted your work.

Reply:
Sounds good. I've opened https://github.com/tc39/proposal-decimal/issues/74 to audit for more of these discrepancies.